primap2/_data_format.py

```diff
@@ -289,9 +289,15 @@ def add_dim(
         terminology: str | None = None,
     ) -> xr.Dataset:
         """
+        TODO: should we rename this to expand_dims to indicate that it does the
+         same as the xarray function, just for primap2 format?
+
         Add a dimension and coordinate to dataset and if a terminology is given also to attrs.
```

Comment:
definitely. That will be less confusion. If we need legacy compatibility, we can keep the `add_dim` as an alias

Reply:
I've done the renaming. An alias is not necessary the legacy of the function is currently very limited, so I can change the calls